doc/develop/api/api_conventions.rst

```diff
@@ -0,0 +1,102 @@
+.. _api_conventions.rst:
+
+API Conventions
+###############
+
+Zephyr provides a few programming interfaces that users can use for creating their applications.
+Zephyr's API space includes the following areas:
+- C programming interface (function prototypes, structures, and macros), provided in C header files
+- Configuration system (Kconfig)
+- Device driver model (Devicetree)
```

Comment:
nitpicking here but I don't think it's fair to describe devicetree as "device driver model", that is more like the struct device and the whole story associated with it, the device tree is more like a hardware description language, should we find a better wording? https://en.wikipedia.org/wiki/Devicetree

Reply:
How about "Hardware description system (Devicetree)"

Reply:
yeah sounds more like it